### DISCUSSION TITLE
elastalert2 help

### DISCUSSION THREAD

**Discussion by peterlymo** [Security-Onion-Solutions/securityonion#8840] [Unsupported Versions]
for past two months i have been working with security onion from installing in vm to deploying in real machine(barrel metal)i have been able to do all initial configuration for NIDS and HIDS and i am able to see all logs but now am in state of dealing with alerts,i was able to ship wazuh alert via email

now the problem is dealing with elastalert2, i could not find a right way to make it happen and i am actually confused on how to make it run, whenever i try i endup broke elastalert docker, any can help help atleast initial configuration?

**Comment by Sahale** [comment]
Hello.

I added creds to `/opt/so/rules/elastalert/smtp_auth_file.txt`:
```
user: so@example.com
password: password_example
```
Then added to rules yaml:
```
...
alert:
  - "email"
email:
  - "email@example.com
smtp_host: "mail.example.com"
smtp_port: 25
from_addr: "so@example.com"
smtp_auth_file: '/opt/elastalert/rules/smtp_auth_file.txt'
...
```

And it works.

**Reply by peterlymo** [reply]
well thanks, what about the rules? you leave it default or there some you added in order to make it work?./////?/

**Reply by Sahale** [reply]
I added this to our custom rules. Didn't try to add to community one.

**Reply by peterlymo** [reply]
If possible can you share that custom rule @Sahale i would like to take it as example so i can reprecate
